src/bootstrap/test.rs

```diff
@@ -565,7 +579,6 @@ impl Step for Miri {
         // miri tests need to know about the stage sysroot
         cargo.env("MIRI_SYSROOT", &miri_sysroot);
         cargo.env("MIRI_HOST_SYSROOT", sysroot);
-        cargo.env("RUSTC_LIB_PATH", builder.rustc_libdir(compiler));
```

Comment:
I remove these because they don't seem to be doing anything. :shrug: Clippy recognizes such a variable, but the current Miri testsuite does not, so it probably just got incorrectly copied at some point.